include/z64audio.h

```diff
@@ -8,9 +8,16 @@
 #define TATUMS_PER_BEAT 48
 
 #define IS_SEQUENCE_CHANNEL_VALID(ptr) ((uintptr_t)(ptr) != (uintptr_t)&gAudioContext.sequenceChannelNone)
+#define SEQ_NUM_CHANNELS 16
 
 #define MAX_CHANNELS_PER_BANK 3
 
+#define MUTE_BEHAVIOR_3 (1 << 3)           // prevent further noteSubEus from playing
```

Comment:
```suggestion
#define MUTE_ACTION_3 (1 << 3)           // prevent further noteSubEus from playing
```

and similar for all these defines? It's a bit shorter.

Reply:
Hmmm this one I want to think about for a bit. Action is a bit of an overused word, I'll have to look over the mute codes again

Reply:
So action makes me think of cause and effect, while behavior is how something acts. These tend to affect how the script interpreter behaves, as opposed to an action that causes something else to happen. I don't know if I'm phrasing that well though.

I definitely want to revisit these flags once more audio is understood, particularly `NoteSubEu` and the `StopSomething` that this flag affects, as we don't really have a full picture to these flags until those two parts are understood.

Reply:
Looking at the dictionary definitions involved
https://www.merriam-webster.com/dictionary/behavior
https://www.merriam-webster.com/dictionary/action

High level actions is a synonym for behaviour. It seems like a behaviour is a collection is actions.

Are these flags making NoteSubEu conduct themselves generally in a specific manner, or is it giving a command to a NoteSubEu?

Reply:
It's definitely not a command or anything, it's just a flag check that runs every frame. Each flag has a different purpose, but for the NoteSubEu one, it looks like this:
```c
if (chan->seqPlayer->muted && (chan->muteBehavior & MUTE_BEHAVIOR_3)) {
    note->noteSubEu.bitField0.finished = true;
}
```

Reply:
Hmm then why is it not mute flags?

Reply:
Yeah, I could go `muteFlags`

Reply:
I'd say mute flags is a better name given the state of documentation. What we know about it currently is it's a flag.

Reply:
This doesn't mean it's the best final name. Just the best with current knowledge.